src/packages/core/components/input-start-node/input-start-node.element.ts

```diff
@@ -0,0 +1,94 @@
+import { UmbInputDocumentElement } from '@umbraco-cms/backoffice/document';
+import { html, customElement, property, css } from '@umbraco-cms/backoffice/external/lit';
+import { FormControlMixin, UUISelectEvent } from '@umbraco-cms/backoffice/external/uui';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbInputMediaElement } from '@umbraco-cms/backoffice/media';
+import { StartNode } from '@umbraco-cms/backoffice/components';
+
+@customElement('umb-input-start-node')
+export class UmbInputStartNodeElement extends FormControlMixin(UmbLitElement) {
+	protected getFormElement() {
+		return undefined;
+	}
+
+	@property()
+	startNodeType?: StartNode['type'];
+
+	@property({ attribute: 'start-node-id' })
+	startNodeId?: string;
+
+	@property({ type: Array })
+	options: Array<Option> = [];
+
+	#onTypeChange(event: UUISelectEvent) {
+		this.startNodeType = event.target.value as StartNode['type'];
+
+		// Clear others
+		this.startNodeId = '';
+		this.dispatchEvent(new CustomEvent('change'));
```

Comment:
You can use the UmbChangeEvent here instead of a custom event.

It will be something like: `this.dispatchEvent(new UmbChangeEvent());` The event class is imported from `@umbraco-cms/backoffice/event`